src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -225,8 +225,8 @@ public override INamedTypeSymbol CreateErrorTypeSymbol(INamespaceOrTypeSymbol co
                 options ?? s_defaultSubmissionOptions,
                 (syntaxTree != null) ? new[] { syntaxTree } : SpecializedCollections.EmptyEnumerable<SyntaxTree>(),
                 references,
-                previousScriptCompilation, 
```

Comment:
Sure, I copied & paste the old version before my commit from VS, maybe VS did this.